lib/tasks/google_charts.rake

```diff
@@ -9,18 +9,24 @@ end
 
 namespace :googlecharts do
   desc "Update google charts javascript dependent files, from latest Builds on google developers website"
-  task :update => [:jsapi, :loader]
+  task :update => [:loader, :jspdf]
   sh "mkdir -p lib/daru/view/adapters/js/googlecharts_js/"
-  task :jsapi do
-    say "Grabbing Core from google jsapi codebase..." do
-      sh "curl -# http://www.google.com/jsapi -L --compressed -o lib/daru/view/adapters/js/googlecharts_js/google_visualr.js"
-    end
-  end
+  # FIXME: Updating jsapi is causing error in IRuby notebook and Googlecharts do not work.
```

Comment:
Open an issue to track this.

Reply:
[#99](https://github.com/SciRuby/daru-view/issues/99)